setup.py

```diff
@@ -1,26 +1,63 @@
+import distutils.cmd
 import os
+import platform
+import shutil
+import subprocess
 import sys
 from distutils.command.clean import clean as _clean
 from distutils.command.clean import log
 from distutils.dir_util import remove_tree
 from multiprocessing import cpu_count
 
 from setuptools import Extension, find_packages, setup
+from setuptools.command.test import test
+
+CYTHON = platform.python_implementation() == "PyPy"
```

Comment:
Why is `CYTHON` True if the implementation is `PyPy`? Isn't that backwards?

Reply:
Yep, it's backwards. It's fixed locally on my uncommited changes.